rector.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+use DrupalFinder\DrupalFinder;
+use Rector\Core\Configuration\Option;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $containerConfigurator->import(__DIR__ .  '/vendor/palantirnet/drupal-rector/config/drupal-8/drupal-8-all-deprecations.php');
+
+    $parameters = $containerConfigurator->parameters();
+
+    $drupalFinder = new DrupalFinder();
+    $drupalFinder->locateRoot(__DIR__);
+    $drupalRoot = $drupalFinder->getDrupalRoot();
+    $parameters->set(Option::AUTOLOAD_PATHS, [
+        $drupalRoot . '/core',
+        $drupalRoot . '/modules',
+        $drupalRoot . '/profiles',
+        $drupalRoot . '/themes'
+    ]);
+
+    $parameters->set(Option::SKIP, ['*/upgrade_status/tests/modules/*']);
+    $parameters->set(Option::FILE_EXTENSIONS, ['php', 'module', 'theme', 'install', 'profile', 'inc', 'engine']);
+    $parameters->set(Option::AUTO_IMPORT_NAMES, true);
+    $parameters->set(Option::IMPORT_SHORT_CLASSES, false);
+    $parameters->set(Option::IMPORT_DOC_BLOCKS, false);
+
+    $parameters->set('drupal_rector_notices_as_comments', true);
```

Comment:
@damontgomery actually this keeps comments. Commits like this fixed commenting, it was one of the first regressions found: https://github.com/palantirnet/drupal-rector/commit/02c49a54bb7a61ac97a20db9987358ca5501128e

Reply:
@mglaman, sorry, this was a bad place to add my question / suggestion. :(.

I meant the comments like `# This will add comments to call out edge cases in replacements.` in https://github.com/palantirnet/drupal-rector/blob/master/rector.yml.